Makefile

```diff
@@ -99,36 +99,64 @@ doc_R_pkg = \
 
 depends = .doc/$(1) .install/$(1) .check/$(1) .test/$(1)
 
-
 ### Rules
 
 .PHONY: all install check test document shiny \
             check_base check_models check_modules 
 
-all: install document
-
+all: install document ## Install all packages and generate documentation
 
-check_base: $(BASE_C) 
-check_models: $(MODELS_C) 
+check_base: $(BASE_C)     ## Check base packages  
+check_models: $(MODELS_C) ## Check model packages   
 
 # Install base first as Modules has a circular dependency on base,
 # and then run a check on modules
-check_modules: $(BASE_I) $(MODULES_C) 
+check_modules: $(BASE_I) $(MODULES_C) ## Install base packages, check module packages
+
+.PHONY: document
+document: ## Generate documentation for all packages or a single package: make document path/to-package
+	@if [ "$(filter-out $@,$(MAKECMDGOALS))" ]; then \
+		PKG="$(filter-out $@,$(MAKECMDGOALS))"; \
+		echo "Documenting package: $$PKG"; \
+		$(call doc_R_pkg, $$PKG); \
+	else \
+		echo "Documenting all packages"; \
+		$(MAKE) $(ALL_PKGS_D) .doc/base/all; \
+	fi
```

Comment:
I agree it would be really nice to have support for `make document path/to/pkg`! But I recommend deferring this to another PR -- what you have here needs more work because it:
* doesn't skip up-to-date packages (notice that `make document base logger && make document base/logger && make document base.logger` runs Roxygen 3 times)
* is noisy (especially the screenful of "...is up to date" when invoked with no arguments)
* should be applied to the `install`, `check`, or `test` goals at the same time -- seems like it'd be pretty confusing to add these piecemeal.